yagb-core/src/emulator/ppu/ppu-cgb.ts

```diff
@@ -41,7 +42,7 @@ const enum stat {
     sourceModeHblank = 0x08,
 }
 
-const SAVESTATE_VERSION = 0x01;
+const SAVESTATE_VERSION = 0x02;
```

Comment:
No need to increment the savestate version just yet --- savestates are currently disabled, and we aren't releasing yet.